### PR TITLE
Changed FIO api urls removed 443 port. Added fioApiRequest log line.

### PR DESCRIPTION
**Pull request by andreyvEze**

### CODE REVIEW

File: src/fio/fioInfo.js [hunk 1]
```diff
@@ -7,25 +7,26 @@ import { imageServerUrl } from '../common/utils'
...
+    'https://fio.greymass.com/v1/',
+    'https://fio.eosusa.news/v1/',
+    'https://fio.eosargentina.io/v1/',
+    'https://fio.cryptolions.io/v1/',
+    'https://fio-mainnet.eosblocksmith.io/v1/',
+    'https://api.fio.currencyhub.io/v1/',
+    'https://fio.eoscannon.io/v1/',
+    'https://fio.eosdublin.io/v1/',
+    'https://fio.eossweden.org/v1/',
+    'https://fio.maltablock.org/v1/'
```
Comment (lukestokes) [inline src/fio/fioInfo.js:29]: Is this node live?

File: src/fio/fioInfo.js [hunk 2]
```diff
@@ -7,25 +7,26 @@ import { imageServerUrl } from '../common/utils'
...
-    'https://fio.eosargentina.io:443/v1/',
-    'https://api.fio.currencyhub.io:443/v1/',
-    'https://fio.eoscannon.io:443/v1/',
-    'https://fio.eossweden.org:443/v1/',
-    'https://fio.maltablock.org:443/v1/',
-    'https://api.fio.eosdetroit.io:443/v1/',
-    'https://fio.eosdublin.io:443/v1/',
-    'https://fio.eosphere.io:443/v1/',
-    'https://fio.cryptolions.io:443/v1/'
+    'https://fio.eosphere.io/v1/',
```
Comment (lukestokes) [inline src/fio/fioInfo.js:13]: Is this node live?

File: src/fio/fioInfo.js [hunk 3]
```diff
@@ -7,25 +7,26 @@ import { imageServerUrl } from '../common/utils'
...
+    'https://fio.zenblocks.io/v1/',
+    'https://api.fio.alohaeos.com/v1/',
+    'https://fio.greymass.com/v1/',
+    'https://fio.eosusa.news/v1/',
+    'https://fio.eosargentina.io/v1/',
+    'https://fio.cryptolions.io/v1/',
+    'https://fio-mainnet.eosblocksmith.io/v1/',
+    'https://api.fio.currencyhub.io/v1/',
+    'https://fio.eoscannon.io/v1/',
+    'https://fio.eosdublin.io/v1/',
```
Comment (lukestokes) [inline src/fio/fioInfo.js:27]: Is this node live?

**Review by lukestokes — COMMENTED** [review]
I'm running this script:
```
#!/bin/bash

urls=( "https://fio.eu.eosamsterdam.net" "https://fio.eosdac.io" "https://fio.eosphere.io" "https://fio.eosrio.io" "https://fio.acherontrading.com" "https://fio.eos.barcelona" "https://api.fio.eosdetroit.io" "https://fio.zenblocks.io" "https://api.fio.alohaeos.com" "https://fio.greymass.com" "https://fio.eosusa.news" "https://fio.eosargentina.io" "https://fio.cryptolions.io" "https://fio-mainnet.eosblocksmith.io" "https://api.fio.currencyhub.io" "https://fio.eoscannon.io" "https://fio.eosdublin.io" "https://fio.eossweden.org" "https://fio.maltablock.org" )

for url in "${urls[@]}"
do
  res=$(curl -s ${url}/v1/node/get_supported_apis | (jq -c '.apis[0]' 2>/dev/null))
  [[ $res = "\"/v1/chain/abi_bin_to_json\"" ]] && alive="OK" || alive="ERROR"
  echo "${url} ${alive}"
done
```
Getting this result:
```
➜  fio.ready-master ./check_nodes.sh 
https://fio.eu.eosamsterdam.net OK
https://fio.eosdac.io OK
https://fio.eosphere.io ERROR
https://fio.eosrio.io OK
https://fio.acherontrading.com OK
https://fio.eos.barcelona OK
https://api.fio.eosdetroit.io OK
https://fio.zenblocks.io OK
https://api.fio.alohaeos.com OK
https://fio.greymass.com OK
https://fio.eosusa.news OK
https://fio.eosargentina.io OK
https://fio.cryptolions.io OK
https://fio-mainnet.eosblocksmith.io OK
https://api.fio.currencyhub.io OK
https://fio.eoscannon.io OK
https://fio.eosdublin.io ERROR
https://fio.eossweden.org OK
https://fio.maltablock.org ERROR
```

File: src/fio/fioInfo.js (hunk 1 above)
Comment (ericbutz) [reply src/fio/fioInfo.js:29]: Confirmed this node is valid.

File: src/fio/fioInfo.js (hunk 2 above)
Comment (ericbutz) [reply src/fio/fioInfo.js:13]: Confirmed this node is valid and working.

File: src/fio/fioInfo.js (hunk 3 above)
Comment (ericbutz) [reply src/fio/fioInfo.js:27]: Confirmed this node is valid and working.

File: src/fio/fioInfo.js [hunk 4]
```diff
@@ -7,25 +7,26 @@ import { imageServerUrl } from '../common/utils'
...
+    'https://fio.eossweden.org/v1/',
```
Comment (ericbutz) [inline src/fio/fioInfo.js:28]: Should be: https://api.fiosweden.org/v1/

**Review by paullinator — APPROVED** [review]

**Pull request merged**